questions/arch.yml

```diff
@@ -0,0 +1,128 @@
+title: What can I do for Arch?
```

Comment:
[questions/archlinux.yml](https://github.com/archlinux/asknot-ng/blob/archlinux/questions/archlinux.yml) already exists, so you can delete this.

Reply:
Okays